### ISSUE TITLE
Corundum not generating in default terraforged world generation

### ISSUE THREAD

**Issue by Epsil-m** [EnigmaticaModpacks/Enigmatica6#4430]
### Modpack Version

1.0.0

### Describe your issue.

I'm at the point where I need corundum but was unable to find it whatsoever. I play on the default E6E world generation type, and spent 2-3 hours looking for some in my home world. I created another world to test it out, went to creative, and flew around again trying to find some (vein miner, and a scanner with a common ores card) and was once again unable to find any. 

I created a standard default minecraft generation world and was able to find some within 5 minutes (same method as before, vein miner with scanner and common ores card). 

I feel like there is some sort of bug or issue between the generation of corundum and the terraforged world generation.

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by MuteTiefling** [comment]
I've certainly seen it spawning in fresh 1.0 worlds. That said, if you're truly unable to find any in the ground, you can get it out of Atum Pyramid chests with some regularity.

**Comment by Zexks** [comment]
I am experiencing this as well. I just ran through 24 pyramids and found none either. Spent about 6 hours today in creative vein mining everything I could to try and find some to no luck.

**Comment by MuteTiefling** [comment]
It's found in caves in the overworld. Vein mining is the worst strategy for finding it. 

We could bump the spawn rate up in atum pyramids too, but did you really loot every chest 13 pyramids and come up empty? That's statistically very improbable.

To be clear, since others have made this mistake, it's in the chests in pyramids. Not the sarcophagus.

**Comment by Zexks** [comment]
Yes I was vein mining the Pyramids and emptying all the chests looking specifically just for the corundum. Got all my god shards and armor sets, But haven't seen a single piece of corundum yet. Oh well might just spawn it in.

**Comment by MuteTiefling** [comment]
Are you getting any custom loot in the chests? Ars gear, aura caches, etc?

**Comment by MuteTiefling** [comment]
I've increased the rates of finding it in atum pyramids. Assuming you guys are even getting the correct loot out of those, that was the intended fallback if you can't find any in world.
